__tests__/hooks/useGroupInfo.test.js

```diff
@@ -621,7 +665,6 @@ describe('useGroupInfo should run correctly', () => {
     expect(result.current.newParticipant).toEqual([
       {user_id: '1234', name: 'elon', user: {name: 'elon'}}
     ]);
-    expect(mockSendMessage).toHaveBeenCalled();
     expect(mockRemoveMember).toHaveBeenCalled();
     expect(mockGroupDispatch).toHaveBeenCalled();
   });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The expectation for `mockSendMessage` to have been called has been removed. This could be due to changes in the implementation of the function being tested where `sendMessage` is no longer called in this scenario.
```diff
-    expect(mockSendMessage).toHaveBeenCalled();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->